app/src/main/java/net/skyscanner/backpack/demo/compose/IconsStory.kt

```diff
@@ -119,7 +119,7 @@ fun IconsStory(modifier: Modifier = Modifier) {
                         contentAlignment = Alignment.Center,
                         modifier = Modifier
                             .border(Dp.Hairline, BpkTheme.colors.line)
-                            .clickable {
+                            .clickableWithRipple {
```

Comment:
renamed this function and made it public, so we don't need to maintain two versions (one in the app and one in backpack)